patches/dompurify+2.3.4.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/node_modules/dompurify/dist/purify.es.js b/node_modules/dompurify/dist/purify.es.js
-index 899e471..5844faf 100644
+index 899e471..414c5d8 100644
```

Comment:
This change is just about reducing the amount of remove/set attribute operations to only remove/set the attribute if needed. It also saves the attributes getting reordered which is something @spocke had mentioned as a concern in a previous Jira.